src/main/resources/templates/probes/results.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" data-layout-decorate="~{layouts/main}">
+<head>
+    <title>Probe Results</title>
+</head>
+<body>
+
+<section data-layout-fragment="content">
+    <h1>Probe Results</h1>
+    <p>This is the result of currently implemented probes.</p>
+    <table class="table">
+        <thead>
+        <tr>
+            <th>ID</th>
+            <th>Result</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr data-th-each="probeResult: ${probeResults}">
+            <td><code data-th-text="${probeResult.key}"></code></td>
+            <td data-th-text="${probeResult.value}"></td>
```

Comment:
> 1. having the total number of plugins known in the result column (like `1800 / 1881` for the `last-commit-date`)

Which one do we like more? This:
* ![image](https://user-images.githubusercontent.com/43273420/199679005-95dd5c6c-ff39-4dd3-bf1b-f39348e70761.png)

OR this?

* ![image](https://user-images.githubusercontent.com/43273420/199682189-d264498d-a1ec-47ff-bdd2-27790132f2e7.png)

Reply:
The first one.